src/uct/tcp/tcp_base.h

```diff
@@ -7,6 +7,10 @@
 #ifndef UCT_TCP_BASE_H
 #define UCT_TCP_BASE_H
 
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
```

Comment:
why needed here?

Reply:
general rure: include config.h into all internal headers
removed

Reply:
the rule as for source files, not for internal headers..